app/views/govuk_publishing_components/components/_summary_list.html.erb

```diff
@@ -68,16 +68,14 @@
     <% if items.any? %>
       <%= tag.dl class: "govuk-summary-list" do %>
         <% items.each do |item| %>
-          <%= tag.div class: "govuk-summary-list__row" do %>
-
+          <%= tag.div class: "govuk-summary-list__row #{"govuk-summary-list__row--no-action" if !(item.fetch(:edit, {}).any? || item.fetch(:delete, {}).any?) }" do %>
+     
```

Comment:
I think there's a typo here. Looks like the class name should `govuk-summary-list__row--no-actions`